src/main/java/uk/ac/ebi/eva/submission/model/SubmissionStatus.java

```diff
@@ -0,0 +1,78 @@
+package uk.ac.ebi.eva.submission.model;
+
+import org.springframework.lang.NonNull;
+
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.Id;
+import javax.persistence.Table;
+import java.time.LocalDateTime;
+import java.util.Objects;
+
+@Entity
+@Table(name = "submission_status")
+public class SubmissionStatus {
+
+    public SubmissionStatus() {
+
+    }
+
+    public SubmissionStatus(String submissionId) {
+        this.submissionId = submissionId;
+    }
+
+    @Id
+    @NonNull
+    @Column(nullable = false, name = "submission_id")
+    private String submissionId;
+
+    @NonNull
+    @Column(nullable = false, columnDefinition = "BOOLEAN DEFAULT FALSE")
+    private boolean completed;
```

Comment:
Is a boolean value fine for this, or do we want a status with pending/successful/failed?

Reply:
Though I guess this isn't a final object model anyway, so probably we can decide later.

Reply:
True. But I think I agree with the having an ENUM here.

Reply:
changed to ENUM